upgrade/UPGRADE-v11.0.0-dev.md

```diff
@@ -843,6 +843,22 @@ There you can find links to upgrade notes for other versions too.
         - collect(Request $request, Response $response, ?Exception $exception = null): void
         + collect(Request $request, Response $response, ?Throwable $exception = null): void
         ```
+- resolve `symfony/translations` deprecations
```

Comment:
add the link to this PR here, please

Reply:
resolved here: https://github.com/shopsys/shopsys/pull/2487/commits/99aa099610cce2da5a0ac5ff3998ed97ab517932